var/spack/repos/builtin/packages/groff/package.py

```diff
@@ -16,41 +16,62 @@ class Groff(AutotoolsPackage, GNUMirrorPackage):
     homepage = "https://www.gnu.org/software/groff/"
     gnu_mirror_path = "groff/groff-1.22.3.tar.gz"
 
+    version('1.22.4', sha256='e78e7b4cb7dec310849004fa88847c44701e8d133b5d4c13057d876c1bad0293')
+    version('1.22.3', sha256='3a48a9d6c97750bfbd535feeb5be0111db6406ddb7bb79fc680809cda6d828a5')
+
     # TODO: add html variant, spack doesn't have netpbm and its too
     # complicated for me to find out at this point in time.
     # See brew scripts for groff for guidance:
     # https://github.com/Homebrew/homebrew-core/blob/master/Formula/groff.rb
     # Seems troublesome...netpbm requires groff?
     variant('pdf', default=True, description='Build the `gropdf` executable.')
+    variant('x', default=False, description='Enable set of graphical options')
+    variant('uchardet', default=True,
+            description='Builds preconv with uchardet library for '
+                        'automatic file encoding detection')
 
     depends_on('gawk',  type='build')
     depends_on('gmake', type='build')
     depends_on('sed',   type='build')
     depends_on('ghostscript', when='+pdf')
+    # iconv is being asked whatever the release
+    depends_on('iconv', type=('build', 'link'))
```

Comment:
No need to specify explicitly the default type:
```suggestion
    depends_on('iconv')
```